### ISSUE TITLE
[GUI] Numbered hotbar

### ISSUE THREAD

**Issue by IdotIcom** [BedrockTweaks/Files#256]
Extra "9" in the inventory button

![IMG_20240811_094733](https://github.com/user-attachments/assets/f50470cb-7c22-4833-9a04-c96b4e22e658)
The number also gets blocked by the items

 - Android 10
 - Minecraft Version: 1.21.2.02

**Additional context**

**Comment by PAPISOP** [comment]
Adding more context:
We can add number over the item render in the hotbar by using json ui, this is already done in resource packs like Better Bedrock

Here's a screenshot of how their numbered hotbar looks:
![image](https://github.com/user-attachments/assets/01080f0a-2d0c-477e-88fb-12fb29fee0c4)
